packages/interaction-viewer/src/interaction-viewer.ts

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable no-param-reassign */
+import { html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import { load } from "data-loader";
+import { select } from "d3";
+import _union from "lodash-es/union";
+import _intersection from "lodash-es/intersection";
+
+import process, { AdjacencyMap, ProcessedData } from "./apiLoader";
+import drawAdjacencyGraph from "./AdjacencyGraph";
+// import drawFilters, { getNameAsHTMLId } from "./filters";
+
+import styles from "./styles";
+import { APIInteractionData } from "./data";
+// eslint-disable-next-line import/no-duplicates
+import InteractionTooltip from "./interaction-tooltip";
+// eslint-disable-next-line import/no-duplicates
+import { FILTER_SELECT } from "./interaction-filters";
+
+// Import additional components
+// eslint-disable-next-line import/no-duplicates
+import "./interaction-filters";
```

Comment:
@aurel-l meant to ask your opinion about this - I wonder if it would be better to have all the WC imports in `index.ts`?

Reply:
Not sure. Originally it was to have the index.ts do the WC registration _and_ re-export, but we decided it was not really useful to expose an unregistered element and have the user register it, and you have the decorator doing the registration in the other file anyway. I think it's important that the entry point remains index.ts, because that's what the tooling will look for as an entry point and we don't want to change the tooling config too much across every package.